source-s3/main.go

```diff
@@ -223,35 +223,40 @@ func main() {
 			"awsAccessKeyId": {
 				"type":        "string",
 				"title":       "AWS Access Key ID",
-				"description": "Part of the AWS credentials that will be used to connect to S3. Required unless the bucket is public and allows anonymous listings and reads."
+				"description": "Part of the AWS credentials that will be used to connect to S3. Required unless the bucket is public and allows anonymous listings and reads.",
+				"order": 0
 			},
 			"awsSecretAccessKey": {
 				"type":        "string",
 				"title":       "AWS Secret Access Key",
 				"description": "Part of the AWS credentials that will be used to connect to S3. Required unless the bucket is public and allows anonymous listings and reads.",
-				"secret":      true
+				"secret":      true,
+				"order":       1
 			},
-			"bucket": {
+			"region": {
 				"type":        "string",
-				"title":       "Bucket",
-				"description": "Name of the S3 bucket"
+				"title":       "AWS Region",
+				"description": "The name of the AWS region where the S3 bucket is located.",
```

Comment:
Might not be easy to see from the diff, but this used to have a default value of `us-east-1` and a suggestion to try using `us-east-1` if you are unsure of what region to use. I find it hard to believe that if a user does not know which region they are capturing from, that blindly trying `us-east-1` is going to be a robust strategy. So I re-worded the description and removed the default.